example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,25 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.ionicfirebaseapp.getwidget_example">
-    <!-- io.flutter.app.FlutterApplication is an android.app.Application that
-         calls FlutterMain.startInitialization(this); in its onCreate method.
-         In most cases you can leave this as-is, but you if you want to provide
-         additional functionality it is fine to subclass or reimplement
-         FlutterApplication and put your custom class here. -->
-    <application
-        android:name="io.flutter.app.FlutterApplication"
-        android:label="GetWidget"
-        android:icon="@mipmap/ic_launcher">
+   <application
+        android:label="example"
```

Comment:
Please compare changes and make sure, it has all proper values

Reply:
@krishnatejakanchi  Okay